assets/product.js

```diff
@@ -266,9 +266,8 @@ function createPlaceholderDiv(id) {
   return div;
 }
 
-const expressCheckoutForm = document.querySelector(
-  '#express-checkout-form'
-);
+const expressCheckoutForm = document.querySelector('#express-checkout-form');
```

Comment:
Please wrap this inside the function that can be called within the express checkout section